understanding/20/contrast-enhanced.html

```diff
@@ -126,7 +126,11 @@ <h2>Intent of Contrast (Enhanced)</h2>
          in charts, graphs, diagrams, and other non-text-based information, which is covered by
          <a href="non-text-contrast">Success Criterion 1.4.11 Non-Text Contrast</a>.         
       </p>
-      
+
+      <div class="note">
+         <p>Text used as part of a logo or logotype is exempted from contrast requirements, under the assumption that logos/logotypes must comply with stricter color choices mandated by corporate identity guidelines. However, this can be problematic when the logo or logotype also acts as a <em>user interface component</em> (such as a link or other interactive control). Some part of the control must meet the 3:1 contrast ratio so users can at least identify that there is a control. Authors could choose a variant of the logo or logotype that has sufficient contrast, include a border with contrast (though this may still not help users identify <em>what</em> the control's label is), or provide an equivalent <em>user interface component</em> which serves the same purpose and does meet the contrast requirements.</p>
```

Comment:
```suggestion
         <p class="wcag21">Text used as part of a logo or logotype is exempted from contrast requirements, under the assumption that logos/logotypes must comply with stricter color choices mandated by corporate identity guidelines. However, this can be problematic when the logo or logotype also acts as a <em>user interface component</em> (such as a link or other interactive control). There are also contrast requirements for <a href="non-text-contrast">Success Criterion 1.4.11 Non-text Contrast</a>, where some part of the control must have contrast, or an accessible alternative.</p>
```

Reply:
yes  this also needs to be worded differently.  This language puts and exception and rationale in a note -- which is not allowable.   If it is an exception the exception needs to be in the provision and the rationale needs to use wording in the provision. 

Eg something like this. 

Note: Text used as part of a logo or logotype falls under the "essential" exemption because logos/logotypes often must comply with stricter color choices mandated by corporate identity guidelines.  

Note:  When the logo or logotype also acts as a <em>user interface component</em> (such as a link or other interactive control) other contrast requirements apply. See  <a href="non-text-contrast">Success Criterion 1.4.11 Non-text Contrast</a>.

and then include the proper note on logos as controls there.